lib/http/service.js

```diff
@@ -58,6 +58,15 @@ module.exports = (container) => {
   const cookieParser = require('cookie-parser');
   service.use(cookieParser());
 
+  service.use((req, res, next) => {
+    if (req.path.startsWith('/sessions/')) {
+      res.set('Cache-Control', 'no-store');
+    } else {
+      res.set('Cache-Control', 'private, max-age=20');
```

Comment:
20 seconds isn't too much?

And for data endpoints like submissions and entities `max-age` should be 0

Reply:
The idea is to have a low `max-age` for all dynamic content so that there is some local caching that comes into play for unusual patterns like immediately refreshing. In general, we don't expect immediate refreshes so this eases load on the server a bit without significantly compromising freshness. We could set it to 0 instead but I don't see a big advantage. Whichever we choose we should apply consistently, I think, since all content is equally dynamic.

Reply:
I think `max-age=0` is likely to provide fewer surprises.

Is immediate refreshing an issue?  I'd guess if it's being done, then it's because of an expectation of changed data...

Reply:
Fair enough, that works for me!